master/usage/routereflector/calico-routereflector.md

```diff
@@ -0,0 +1,277 @@
+---
+title: 'Calico BIRD Route Reflector container'
+---
+
+This guide discusses the calico/routereflector image: a container image that
+packages up the `BIRD` BGP daemon along with the `confd` templating engine to
+provide a simple route reflector image which may be used for scaled-out Calico
+deployments.
+
+The image is currently experimental due to some key limitations discussed below.
+However, it may provide a useful framework for future development.
+
+#### Known limitations
+
+-  There is no mechanism to explicitly configure peers with the Route Reflector,
+   thus is it not possible to easily peer the route reflector with the fabric.
```

Comment:
"with the fabric" is a bit vague.

This might be better phrased as "This container will automatically peer with Calico, but has no mechanism to configure peerings with non-calico BGP speakers (e.g physical infrastructure)"

WDYT?